ResourceStatusSystem/Agent/SiteInspectorAgent.py

```diff
@@ -0,0 +1,183 @@
+# $HeadURL:  $
+""" SiteInspectorAgent
+
+  This agent inspect Sites, and evaluates policies that apply.
+
+"""
+
+import datetime
+import math
+import Queue
+
+from DIRAC                                                      import S_ERROR, S_OK
+from DIRAC.Core.Base.AgentModule                                import AgentModule
+from DIRAC.Core.Utilities.ThreadPool                            import ThreadPool
+from DIRAC.ResourceStatusSystem.Client.SiteStatus               import SiteStatus
+from DIRAC.ResourceStatusSystem.PolicySystem.PEP                import PEP
+from DIRAC.ResourceStatusSystem.Utilities                       import Utils
+ResourceManagementClient = getattr(Utils.voimport( 'DIRAC.ResourceStatusSystem.Client.ResourceManagementClient' ), 'ResourceManagementClient')
+
+__RCSID__  = '$Id$'
+AGENT_NAME = 'ResourceStatus/ElementInspectorAgent'
+
+class SiteInspectorAgent( AgentModule ):
+  """ ElementInspectorAgent
+
+  The ElementInspector agent is a generic agent used to check the elements
+  of one of the elementTypes ( e.g. Site, Resource, Node ).
+
+  This Agent takes care of the Elements. In order to do so, it gathers
+  the eligible ones and then evaluates their statuses with the PEP.
```

Comment:
This docstring needs an update, right now it is only a copy/paste from the ElementInspectorAgent